passes/cmds/select.cc

```diff
@@ -987,7 +987,7 @@ struct SelectPass : public Pass {
 		log("list of selected objects.\n");
 		log("\n");
 		log("Note that many commands support an optional [selection] argument that can be\n");
-		log("used to YS_OVERRIDE the global selection for the command. The syntax of this\n");
+		log("used to override the global selection for the command. The syntax of this\n");
```

Comment:
Hahaha!  Looks like this was collateral damage when we switched all users of the C++ keyword `overwrite` to the `YS_OVERWRITE` macro.

Reply:
Yes. I was greatly amused by this one.

Reply:
https://www.newscientist.com/letter/mg16221857-700-ocralomania/